yarn.lock

```diff
@@ -1722,6 +1722,18 @@
     tslib "^2.4.0"
     undici "^5.5.1"
 
+"@elastic/transport@^8.3.2":
+  version "8.3.2"
```

Comment:
__optional nit:__ we can join `@elastic/transport` declarations, _but_ it's 8.9 branch that will go out of support soon and it's unlikely it becomes a problem before that, and I don't want you to wait another CI run just for that, so feel free to keep it as is :slightly_smiling_face:

Reply:
yeah... I didn't care much about it here because I had https://github.com/elastic/kibana/pull/164076 waiting for this one.

https://github.com/elastic/kibana/pull/164076 does the merge 😇